README.md

```diff
@@ -212,7 +212,7 @@ For [OpenSUSE, CentOS, Fedora, RHEL and SLE](https://build.opensuse.org/package/
 Kai-Uwe Behrmann kindly provides packages [for download](http://software.opensuse.org/download.html?project=home%3Abekun%3Adevel&package=libelektra4).
 For Debian wheezy and jessie amd64 we provide latest builds. See build server below.
 
-If there are no packages available for your distribution, see the
+If there are no packages available for your distribution, or if you are using a different operating system like macOS, see the
```

Comment:
ah yeah, i added it to the package list. 
Encountered a small issue with the brew package https://github.com/ElektraInitiative/libelektra/issues/1243 which shouldnt be a problem though for normal users.

I was not sure how to control which optional components of elektra shall be built using the homebrew package but after a small study on the homebrew issue https://github.com/ElektraInitiative/libelektra/issues/531 i found out that i can see the parameters via brew info elektra. Is it worth mentioning this in the readme of our tap?